kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/FitbitRestSourceConnectorConfig.java

```diff
@@ -115,18 +111,21 @@
   private static final String FITBIT_ACTIVITY_LOG_TOPIC_DEFAULT = "connect_fitbit_activity_log";
   private static final String FITBIT_ACTIVITY_LOG_TOPIC_DISPLAY = "Activity log topic";
 
-  private final UserRepository userRepository;
+  private final List<UserRepository> userRepositories;
   private final Headers clientCredentials;
 
   @SuppressWarnings("unchecked")
   public FitbitRestSourceConnectorConfig(ConfigDef config, Map<String, String> parsedConfig, boolean doLog) {
     super(config, parsedConfig, doLog);
+    userRepositories = new ArrayList<>();
 
     try {
-      userRepository = ((Class<? extends UserRepository>)
-          getClass(FITBIT_USER_REPOSITORY_CONFIG)).getDeclaredConstructor().newInstance();
+      for(String c: getList(FITBIT_USER_REPOSITORIES_CONFIG)) {
+        userRepositories.add(
+            ((Class<? extends UserRepository>) Class.forName(c)).getDeclaredConstructor().newInstance());
+      }
     } catch (IllegalAccessException | InstantiationException
-        | InvocationTargetException | NoSuchMethodException e) {
+        | InvocationTargetException | NoSuchMethodException | ClassNotFoundException e) {
       throw new ConnectException("Invalid class for: " + SOURCE_PAYLOAD_CONVERTER_CONFIG, e);
```

Comment:
Shouldn't this be `FITBIT_USER_REPOSITORIES_CONFIG ` ?

Reply:
Yes. good spot. i did not notice that. Must have been a typo from previous commits.